wavefront_opentracing_sdk/span.py

```diff
@@ -92,6 +94,25 @@ def set_tag(self, key, value):
                         force_sampling)
         return self
 
+    def log_kv(self, key_values, timestamp=None):
```

Comment:
You could use `**kwargs` instead of `key_values` here

Reply:
Wanna to following opentracing python here: https://github.com/opentracing/opentracing-python/blob/master/opentracing/span.py#L145

Reply:
Gotcha!